src/room.js

```diff
@@ -1,7 +1,7 @@
 class Room {
-
   constructor (roomname) {
     this.roomname = roomname;
+    this.pin = Math.floor(Math.random() * 10000);
```

Comment:
To get a number between 1000-9000 (four digit pin always), perhaps this solution;
```javascript
Math.floor(Math.random() * 8000) + 1000
```